tests/unit_tests/test_csv_splitter_unit.py

```diff
@@ -21,52 +22,37 @@ def temp_file() -> StringGeneratorType:
 temp_csv_file = create_temp_file_fixture(file_extension="csv")
 temp_txt_file = create_temp_file_fixture(file_extension="txt")
 
-
-@pytest.fixture
-def create_files_from_chunks():
-    create_files(chunks=CHUNKS, output_file_name="out")
-    yield
-    os.remove("out_1.csv")
-    os.remove("out_2.csv")
+valid_row_limits = random.randint(2, 7)
```

Comment:
We don't like to use `random` in our tests. We want the test behavior to be predictable.